src/defaultsettings.cpp

```diff
@@ -470,6 +472,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("fps_max_unfocused", "10");
 	settings->setDefault("max_objects_per_block", "20");
 	settings->setDefault("sqlite_synchronous", "1");
+	settings->setDefault("map_compression_level", "-1");
```

Comment:
We might want to default net_compression_level for 3 for android, since these are all local games.
Also would be good to default it to 3 for any single player games.

Reply:
Yeah that'd be indeed be a very good idea for singleplayer.
In theory you could even go to level=0 but putting the netcode under more stress isn't a good idea even over localhost.

Reply:
Actually... I can't know whether it's singleplayer at this place, though. Can I?
For Android definitely.